test/helpers/position_assertions.h

```diff
@@ -10,18 +10,6 @@
 namespace sorbet::test {
 using namespace sorbet::realmain::lsp;
 
-// Compares the two errors. Returns -1 if `a` comes before `b`, 1 if `b` comes before `a`, and 0 if they are equivalent.
-// Compares filenames, then ranges, and then compares messages in the event of a tie.
-int errorComparison(std::string_view aFilename, const Range &a, std::string_view aMessage, std::string_view bFilename,
```

Comment:
where these just unused?

Reply:
They were only used in position_assertions.cc, where they are now declared and implemented at the same time.